varats/varats/tools/driver_casestudy.py

```diff
@@ -304,6 +304,13 @@ def __create_cleanup_parser(sub_parsers: _SubParsersAction) -> None:
         choices=["old", "error", "regex"],
         type=str
     )
+    cleanup_parser.add_argument(
+        "--filter",
```

Comment:
In `vara-cs status` we called it `--filter-regex`, using the same here would make the option consistent.
```suggestion
        "--filter-regex",
```